### PR TITLE
Reduced fees for partner projects

### PR DESCRIPTION
**Pull request by DonDuala**

### CODE REVIEW

**Review by brittcyr — APPROVED** [review]

**Comment by DonDuala** [comment]
Ok to merge with the failed tests?

**Comment by brittcyr** [comment]
> Ok to merge with the failed tests?

soteria is expected to fail.
just run

cargo fmt

to fix lint

**Comment by DonDuala** [comment]
can you pls handle deployment

File: programs/staking-options/src/instructions/mod.rs [hunk 1]
```diff
@@ -6,7 +6,6 @@ pub mod issue;
 pub mod name_token;
 pub mod withdraw;
 
-pub use add_tokens::*;
```
Comment (DonDuala) [inline programs/staking-options/src/instructions/mod.rs:9]: @brittcyr this ok? Seems weird this wasn't caught before?

Comment (brittcyr) [reply programs/staking-options/src/instructions/mod.rs:9]: doesnt matter

**Pull request merged**